module/english_cambridge.py

```diff
@@ -43,9 +43,9 @@ def LookUp(word, data):
             english = tabEntry.find('div', class_='tabs__content on', attrs={"data-tab": "ds-business-english"})
 
     partOfSpeech = english.find_all('div', class_='entry-body__el clrd js-share-holder')
-    sound = partOfSpeech[0].find('span', attrs={'data-src-mp3':True})['data-src-mp3']
+    sound = partOfSpeech[0].find('span', attrs={'data-src-mp3':True})
     if(sound is not None):
-        wget.download(sound, out=download_dir+"Py_"+word+".mp3")
+        wget.download(sound['data-src-mp3'], out=download_dir+"Py_"+word+".mp3")
```

Comment:
what will happen if the sound doesn't contains 'data-src-mp3' key ?

Reply:
I haven't faced this problem yet XD.
Maybe I can add one more expression to prevent this problem from accident.